lib/motors/pid_controller.hpp

```diff
@@ -0,0 +1,47 @@
+#include "core/types.hpp"
+#pragma once
+
+namespace hyped::motors {
+class PidController {
+ public:
+  PidController(
+
```

Comment:
whitespace, i'm suprised clang-format is alright with this i thought it would change it ngl